api/models/FallSport.js

```diff
@@ -6,14 +6,30 @@
  */
 module.exports = {
     attributes: {
-        name: { type: "string", required: true, unique: true }
+        name: { type: "string", required: true, unique: true, },
+        status: {type: "string", required: true, isIn: ["Yes", "No"]},
```

Comment:
I think `status` is an odd name for a field with Yes/No values. How about `active`?